commit 8911a632c3cdee700d43d71dd391d5ae81b1dbcb
Author: Matt Flax <flatmax@flatmax.org>
Date:   Wed Mar 8 21:13:24 2017 +1100

    ASoC: bcm2835_i2s.c: relax the ch2 register setting for 8 channels
    
    This patch allows ch2 registers to be set for 8 channels of audio.

diff --git a/sound/soc/bcm/bcm2835-i2s.c b/sound/soc/bcm/bcm2835-i2s.c
index 6ba20498202e..c8dd065aea84 100644
--- a/sound/soc/bcm/bcm2835-i2s.c
+++ b/sound/soc/bcm/bcm2835-i2s.c
@@ -239,6 +239,7 @@ static int bcm2835_i2s_hw_params(struct snd_pcm_substream *substream,
 	unsigned int ch1pos, ch2pos, mode, format;
 	uint32_t csreg;
 
+
 	/*
 	 * If a stream is already enabled,
 	 * the registers are already set properly.
@@ -312,6 +313,7 @@ static int bcm2835_i2s_hw_params(struct snd_pcm_substream *substream,
 
 	switch (params_channels(params)) {
 	case 2:
+	case 8:
 		format = BCM2835_I2S_CH1(format) | BCM2835_I2S_CH2(format);
 		format |= BCM2835_I2S_CH1(BCM2835_I2S_CHPOS(ch1pos));
 		format |= BCM2835_I2S_CH2(BCM2835_I2S_CHPOS(ch2pos));
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1204_8911a632c3cdee700d43d71dd391d5ae81b1dbcb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1204_8911a632c3cdee700d43d71dd391d5ae81b1dbcb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
